app/code/BVMyBlog/Blog/Block/Post.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Block;
+
+/**
+ * Class Post
+ *
+ * Short description
```

Comment:
1. Description should be descriptive. 
2. Remove descriptions like 'Class ClassName'